apps/src/templates/lessonOverview/LessonOverview.jsx

```diff
@@ -62,14 +63,14 @@ class LessonOverview extends Component {
     );
   }
 
-  recordAndNavigateToPdf = (e, firehoseKey, url) => {
-    // Prevent navigation to url until callback
-    e.preventDefault();
+  recordAndHandleResource = (e, firehoseKey, action, url = null) => {
+    e.preventDefault(); // Prevent navigation to url until callback
+    const event = action === 'navigate' ? 'open-pdf' : 'print-from-browser';
```

Comment:
Optional: Do you think it makes sense to put `'navigate'` , `'open-pdf`' etc in constants at the top of this file instead of using these strings (ex. `const OPEN_PDF = 'open-pdf'`)

Reply:
Probably.
It's a large component and I thought about making a separate react component, but I was not sure what's the teacher-tools team's preference.
I will go ahead and move those variables to the top. 
I will also add some screenshots to the PR description.